src/main/resources/db/migration/V6.5.0.1.sql

```diff
@@ -0,0 +1,10 @@
+-- Fixes File Access Requests when upgrading from Dataverse 6.0
+-- See: https://github.com/IQSS/dataverse/issues/10714
+DELETE FROM fileaccessrequests
+WHERE creation_time <> (SELECT MIN(creation_time)
+                        FROM fileaccessrequests far2
+                        WHERE far2.datafile_id  = fileaccessrequests.datafile_id
+                          AND far2.authenticated_user_id  = fileaccessrequests.authenticated_user_id
+                          AND far2.request_state is NULL);
+
+UPDATE fileaccessrequests SET request_state='CREATED' WHERE request_state is NULL;
```

Comment:
Did you consider putting a DB constraint on as Jim suggested?

Reply:
@qqmyers Could you add your suggestion here please?